src/pretix/plugins/sendmail/models.py

```diff
@@ -205,6 +219,12 @@ class Rule(models.Model, LoggingMixin):
         (BOTH, _('Both (all order contact addresses and all attendee email addresses)'))
     ]
 
+    CHECK_IN_STATUS_CHOICES = [
+        ("", _("Everyone")),
```

Comment:
Does this need to be
```suggestion
        (None, _("Everyone")),
```
as well so None is stored when I select it in the form?

Reply:
Yes, it does